package.json

```diff
@@ -1,16 +1,29 @@
 {
   "name": "react-financial-portal",
-  "version": "0.1.0",
   "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc -b && vite build",
```

Comment:
**suggestion**: Consider adding a script for running tests if testing is part of the development process.